submissions/nadsatt/friends-app/modules/header.js

```diff
@@ -0,0 +1,61 @@
+import svgs from './../modules/svgs.js';
+
+export class Header {
+    constructor(){
+        this.defineElement();
+        this.insertElement();
+
+        this.defineSoundButton();
+        this.defineSoundButtonClickHandler();
+    }
+
+    defineElement(){
+        const HTML =
+            `<h2 class="header__heading">Friends App</h2>
+             <div class="header__sound-button">
+                <span class="header__sound-icon">${svgs.play}</span>
+                <span class="header__text">play</span>
+             </div>`;
+
+        this.element = document.createElement('header');
+        this.element.classList.add('header');
+        this.element.insertAdjacentHTML('afterbegin', HTML);
+    }
+
+    insertElement(){
```

Comment:
@zonzujiro  Because, as i understand, function name should reflect what function do. 

I mean, when i use method with name **define** i don't expect that this method can do anything **except defining element** (like inserting this element somewhere). 

I had some uncertainty about this, so it would be nice if you clarify it for me.

Reply:
Ok. Thx for an explanation. You are right - functions should be responsible for one thing. But creation and inserting the element into DOM is one simple operation, so no need for separation here. It can be in one function.

My main idea is that you have created a lot of entities, which you probably don't need.  Occam's razor :)